public/blog/first-ever-truffle-retreat.md

```diff
@@ -0,0 +1,123 @@
+
+This turned out to be an interesting exercise. As the leader of the team you want to say “we’re doing great”, and not diminish anyone’s efforts -- because, it’s true, we’ve come far for a small team. However, we need to know where we can improve and how different areas of our company, teamwork and products stack up to the rest. In general, we all seemed to agree on the results: Branding and growth have been stellar (e.g., we have a good product that people like to use), but more external communication -- like our technical leadership, our relationship with the Ethereum Foundation, and relationships with some of our VIP users -- could use some improvement. Though these results were mostly subjective judgements, the conversations that resulted helped inform discussions about our roles and future roadmap. 
+
+### Roles & Responsibilities
+
+The Truffle team had already doubled in size in 2017, and right before the retreat two new team members -- Ben Burns and Mike Pumphrey -- had accepted offers (welcome!). So we knew we had some work to do regarding defining our roles and how we work together as a team.
+
+Before the retreat, I had met separately with Nick, Josh, and John to evaluate how they felt about their current roles at Truffle. We each talked about what they wanted to do both within the team and professionally, and how I could help them get there. Then we used this time at the retreat to reorganize the work so that it aligns with each team member’s personal goals as well as the group’s goals. We created a new org chart and chain of command, including the new team members we knew were coming on. 
+
+We decided that Josh and John would share design responsibilities for our website, logo design and general aesthetics for all of our products. Ben, who will start in October, would take on the internals of the TestRPC (now [Ganache](http://truffleframework.com/ganache/)) and related products ensuring we stay up to date with the Ethereum ecosystem. And Mike would be coming on board to wrangle all of our documentation and learning content, eventually molding it into tailored courses we’d offer to individuals and businesses. 
+
+That just left Nick and I. Though we both knew Nick would work on the internals of Truffle when he was hired, we didn’t know the extent of the work required; in addition to improving Truffle, Nick would be burdened with the arduous task of softly and lovingly ripping my baby from my arms. That is, Truffle’s my baby, and I’ve been nurturing it from day one. In order to build a more functional and growing team, it became clear I’d have to let go of the internal implementation details and focus more on our business and our product, leaving Nick to explore the tool’s technical way forward.
+
+This transition has been a great growing experience already, only a few weeks after the retreat itself. We’ll continue to evaluate this structure over the next six months to see how it works out. 
```

Comment:
Might rephrase as it's been more than a few weeks.

Reply:
✔️